experimental/iterators/lib/Conversion/IteratorsToLLVM/IteratorsToLLVM.cpp

```diff
@@ -1249,62 +1249,59 @@ static Value convert(IteratorOpInterface op, ValueRange operands,
   return buildStateCreation(op, builder, stateType, operands);
 }
 
-/// Converts the given sink to LLVM using the converted operands from the root
-/// iterator. The current sink consumes the root iterator and prints each
-/// element it produces. Pseudo code:
+/// Converts the given sink to LLVM using the converted input iterator. The
+/// current sink consumes the input iterator and prints each element it
+/// produces. Pseudo code:
 ///
-/// rootIterator->Open()
-/// while (nextTuple = rootIterator->Next())
+/// input->Open()
+/// while (nextTuple = input->Next())
 ///   print(nextTuple)
-/// rootIterator->Close()
+/// input->Close()
 ///
 /// Possible result:
 ///
-/// %2 = ... // initialize state of root iterator
+/// %2 = ... // initialize state of input iterator
 /// %3 = call @iterators.reduce.open.1(%2) :
```

Comment:
Side note: you can specify SSA result names for specific ops.
E.g. %3 could be called "%opened"

Reply:
Note to self: this can be done with the [`OpAsmOpInterface`](https://github.com/llvm/llvm-project/blob/5ab6ef7/mlir/include/mlir/IR/OpAsmInterface.td#L24).